changelogs/fragments/71279-skip-conflict-check.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+- setup.py - Skip doing conflict checks for ``sdist`` and ``egg_info`` commands
+  (https://github.com/ansible/ansible/pull/71279)
```

```suggestion
  (https://github.com/ansible/ansible/pull/71310)
```
I think it might be less confusing to just link to this PR (#71310) even though that one inspired it.